README.md

```diff
@@ -6,9 +6,16 @@ To learn more, see the [SnarkyJS reference](https://docs.minaprotocol.com/en/zka
 
 For a list of changes between versions, see the [CHANGELOG.md](https://github.com/o1-labs/snarkyjs/blob/main/CHANGELOG.md).
 
-## Community packages
+## Community Packages
 
-A list of community-maintained packages is being collected. To include your package, see the [Contributing guidelines](https://github.com/o1-labs/snarkyjs/blob/main/CONTRIBUTING.md#creating-high-quality-community-packages).
+To include your package, see the [Contributing guidelines](https://github.com/o1-labs/snarkyjs/blob/main/CONTRIBUTING.md#creating-high-quality-community-packages).
+
+Thank you to our community members who contribute packages that can be installed from [npm](https://www.npmjs.com/) and used alongside SnarkyJS. 
+
+Here's our official list of community-maintained packages:
+
+- SnarkyJS Elgamal
+  - A partial homomorphic encryption scheme. Contributed by https://github.com/Trivo25/.
```

Comment:
This should have links to github and npm, see the discussion on https://github.com/o1-labs/snarkyjs/pull/856

Reply:
Also, I'm arguing that the package spelling should be consistent with the github repo / npm package, so `snarkyjs-elgamal` in this case